iosTests/package.json

```diff
@@ -10,9 +10,9 @@
     },
     "dependencies": {
         "create-react-class": "^15.7.0",
-        "react": "17.0.2",
-        "react-native": "0.67.1",
-        "react-native-force": "git+https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git#dev"
+        "react": "18.1.0",
+        "react-native": "0.70.1",
+        "react-native-force": "git+https://github.com/wmathurin/SalesforceMobileSDK-ReactNative.git#rn0701"
```

Comment:
Pointing to `wmathurin/SalesforceMobileSDK-ReactNative.git#rn0701` for now allow CI to succeed.
Will point to `forcedotcom/SalesforceMobileSDK-ReactNative.git#dev` before merging